### PR TITLE
Update rails helper with requires from support

### PR DESCRIPTION
**Pull request by AleksSenkou**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -25,28 +25,36 @@ gem 'mini_magick'
 gem 'devise', '~> 3.5'
 gem 'omniauth-google-oauth2'
 
-group :development, :test do
+# helping tools
+gem 'require_all'
```
Comment (roman-dubrovsky) [inline Gemfile]: @AleksSenkou I don't understand well why did you use this gem?

File: Gemfile [hunk 2]
```diff
@@ -25,28 +25,36 @@ gem 'mini_magick'
...
 
-group :development do
-  gem 'thin'
-  gem 'bullet'
-  gem 'quiet_assets'
-  gem 'web-console'
-  gem 'rails_real_favicon'
-  gem 'capistrano-db-tasks', require: false
+group :test do
+  gem 'rspec-rails', '~> 3.5'
```
Comment (roman-dubrovsky) [inline Gemfile]: Gem's contributors advises to add this gem to `:test, :development` group

https://github.com/rspec/rspec-rails#installation

File: Gemfile (hunk 1 above)
Comment (roman-dubrovsky) [inline Gemfile]: oh, I've seen it below

Comment (roman-dubrovsky) [inline Gemfile]: @AleksSenkou also I think that this comment is unnecessary in Gemfile

Comment (AleksSenkou) [inline Gemfile]: @roman-dubrovsky I like modules in Gemfile. Those comments divide gems to logical blocks

File: Gemfile (hunk 2 above)
Comment (AleksSenkou) [inline Gemfile]: @roman-dubrovsky good point. thank you
do you know why do we need it for both envs?

File: Gemfile (hunk 1 above)
Comment (roman-dubrovsky) [inline Gemfile]: @AleksSenkou I don't know, it looks a little bit strange

File: Gemfile (hunk 2 above)
Comment (roman-dubrovsky) [inline Gemfile]: @AleksSenkou no, I don't

**Pull request merged**